### DISCUSSION TITLE
fix: I want it to retain scrolling.

### DISCUSSION THREAD

**Discussion by kurages** [community/community#39491] [Other Feature Feedback, Questions, & Ideas] [answered]
I don't want to have to move to the top every time I add a repository to the list.

**Comment by jfullstackdev** [comment]
I just added a new list and it appears on top,

![image](https://user-images.githubusercontent.com/47092464/202838146-5a9dd906-3dd8-4adf-af79-41ef5b4c68b6.png)

that is I think what you mean and it's helpful, why would you not want that to appear on top ? 

besides you can sort it from time to time

**Reply by atlslscsrv-app** [reply]
That's very kind of you. If no one has told you your appreciated lately, your appreciated 👍

**Comment by twiddler** [comment, marked as answer]
I think @kurages is talking about the Android app. See #20353.
